README.md

```diff
@@ -11,106 +11,54 @@ This tool may be especially useful for development and testing.
 
 ## Installation
 
-### Using install script
+The binary will be installed into `$GOPATH/bin`. Be sure that `$GOPATH` is defined in your environment,
+and that `$GOPATH/bin` is in your `$PATH`.
 
-This is the preferred way. Does not require golang to be installed on the system.
-
-To download a binary for the latest release, run:
-
-```sh
-curl -sSfL https://raw.githubusercontent.com/ava-labs/avalanche-network-runner/main/scripts/install.sh | sh -s
-```
-
-The binary will be installed inside the `./bin` directory (relative to where the install command was run).
-
-_Downloading binaries from the Github UI will cause permission errors on Mac._
-
-To add the binary to your path, run
-
-```sh
-cd bin
-export PATH=$PWD:$PATH
-```
-
-To add it to your path permanently, add an export command to your shell initialization script (ex: .bashrc).
-
-#### Installing in Custom Location
-
-To download the binary into a specific directory, run:
-
-```sh
-curl -sSfL https://raw.githubusercontent.com/ava-labs/avalanche-network-runner/main/scripts/install.sh | sh -s -- -b <relative directory>
-```
-
-### Install using golang
-
-Requires golang to be installed on the system ([https://go.dev/doc/install](https://go.dev/doc/install)).
+To install the latest binary locally, run:
 
 ```sh
-go install github.com/ava-labs/avalanche-network-runner@latest
+curl -sSfL https://raw.githubusercontent.com/ava-labs/avalanche-network-runner/main/scripts/install.sh | sh -s -- -b $GOPATH/bin
 ```
 
-After that, the `avalanche-network-runner` binary should be present under the `$HOME/go/bin/` directory. Consider adding this directory to the `PATH` environment variable.
-
-### Install by release download
+## Build from source code
 
-Does not require golang to be installed on the system.
+This is only needed by advanced users who want to modify or test Avalanche Network Runner in specific ways.
 
-Download the desired distribution from [https://github.com/ava-labs/avalanche-network-runner/releases](https://github.com/ava-labs/avalanche-network-runner/releases).
-
-Uncompress and locate where is convenient. Consider adding the target bin directory to the `PATH` environment variable.
-
-### Install from source code and execute tests
-
-#### Download
+### Download
 
 ```sh
 git clone https://github.com/ava-labs/avalanche-network-runner.git
 ```
 
-#### Install
+### Build
 
 From inside the cloned directory:
 
 ```sh
-go install
+./scripts/build.sh
 ```
 
-After that, `avalanche-network-runner` binary should be present under `$HOME/go/bin/` directory. Consider adding this directory to the `PATH` environment variable.
+After that, `avalanche-network-runner` binary should be present under the `./bin/` directory, inside the cloned one. The user may consider to add this directory to the `$PATH` environment variable.
 
```

Comment:
Can we add the command to add directory to PATH environment variable here? Thanks